src/main/java/me/mrCookieSlime/Slimefun/Events/DisenchanterEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class DisenchanterEvent extends Event implements Cancellable {
+	
+	private static final HandlerList handlers = new HandlerList();
+
+	private final ItemStack sfItem;
```

Comment:
This field should just be called "item" since it represents an ItemStack and not a Slimefunitem.
It's misleadingly named.